ForageBuddy/NameReader.cs

```diff
@@ -12,11 +12,11 @@ public static class NameReader
     {
         private const int DutyReportScalar = 4;
 
-        private const int BrigtnessThreshold = 100;
+        private const int BrightnessThreshold = 100;
```

Comment:
Can this be above the method it's used in?

Reply:
I think it makes more sense to keep them at the private consts at the top of this class, as the class is quite small, and the values reasonably well named.